### DISCUSSION TITLE
Elastic driving force for grain growth: lack of documentation of Euler angle file

### DISCUSSION THREAD

**Discussion by jjmarzia** [idaholab/moose#22448] [Q&A Modules: Phase field] [answered]
Hi all, I'm studying the case of [poly_grain_growth_2D_eldrforce.i](https://github.com/idaholab/moose/blob/next/modules/combined/examples/phase_field-mechanics/poly_grain_growth_2D_eldrforce.i). Line 36 of this example requires an associated .tex file containing a set of Euler angles corresponding to each grain number, called **grn_36_rand_2D.tex**, but I have not seen documentation of this file. I was wondering if this was somewhere available. 

Thank you!

**Comment by GiudGiud** [comment, marked as answer]
Hello
This object has indeed not been documented
https://mooseframework.inl.gov/source/userobjects/EulerAngleFileReader.html

You can try to mimick the syntax used in the list of input files (and their associated angles files) shown by this documentation page.
If you run into any issues, @permcody or @amjokisaari will be able to help you

Guillaume
